docs/docs/references/users/README.md

```diff
@@ -214,3 +214,69 @@ Updating a user's _custom data_ will overwrite (not merge) its original content
 :::
 
 ### Identities
+
+_Identities_ contain the user info retrieved from social sign-in
+(i.e., sign-in with a social connector).
+
+The user info varies by provider,
+but it will typically include the following:
+
+- Name of this provider, such as lowercase "facebook", "google", or "wechat"
+- User's unique identifier for this provider
+- User's name
+- User's verified email
+- User's avatar
+
+The user may link their account to multiple social identity providers via social sign-in, such as Facebook, Google, and Wechat;
```

Comment:
use `WeChat` to express the meaning of "wechat" across all UI and docs.

Reply:
Fixed